python/lsst/ctrl/mpexec/cli/script/report.py

```diff
@@ -106,16 +107,200 @@ def report(
         datasets.add_column(data_products, index=0, name="DatasetType")
         quanta.pprint_all()
         print("\n")
-        if show_errors:
+        if not brief:
             pprint.pprint(error_summary)
             print("\n")
-        else:
-            assert qgraph.metadata is not None, "Saved QGs always have metadata."
-            collection = qgraph.metadata["output_run"]
-            collection = str(collection)
-            run_name = collection.split("/")[-1]
-            with open(f"{run_name}_err.yaml", "w") as stream:
-                yaml.safe_dump(error_summary, stream)
         datasets.pprint_all()
     else:
         report.write_summary_yaml(butler, full_output_filename, do_store_logs=logs)
+
+
+def report_v2(
+    butler_config: str,
+    qgraph_uris: Sequence[str],
+    collections: Sequence[str] | None,
+    where: str,
+    full_output_filename: str | None,
+    logs: bool = True,
+    brief: bool = False,
+    curse_failed_logs: bool = False,
+) -> None:
+    """Summarize the state of executed quantum graph(s), with counts of failed,
+    successful and expected quanta, as well as counts of output datasets and
+    their visible/shadowed states. Analyze one or more attempts at the same
+    processing on the same dataquery-identified "group" and resolve recoveries
+    and persistent failures. Identify mismatch errors between groups.
+
+    Parameters
+    ----------
+    butler_config : `str`
+            The Butler used for this report. This should match the Butler used
+            for the run associated with the executed quantum graph.
+    qgraph_uris : `Sequence[str]`
```

Comment:
I think the formatting of this is different, but cannot find exact example.  See https://developer.lsst.io/python/numpydoc.html#sequence-types for list example.